src/game/common/system/geometry.cpp

```diff
@@ -334,6 +350,11 @@ void GeometryInfo::Clip_Point_To_Footprint(const Coord3D &pos, Coord3D &point) c
     }
 }
 
+inline bool Is_Point_In_Range(float min, float val, float max)
+{
+    return min <= val && val <= max;
```

Comment:
That's a very weird way of writing this. Could you make it val > min && val <= max. Also are you sure it's exclusive of min and inclusive of max. Again a bit weird.